packages/cf.js/src/app-factory.ts

```diff
@@ -1,5 +1,42 @@
-import { AppDefinition } from "./structs";
+import {
+  NodeMessageType,
+  NodeProposeInstallData
+} from "@counterfactual/node-provider";
+import { BigNumber, BigNumberish } from "ethers/utils";
+
+import { AppDefinition, BlockchainAsset } from "./app-types";
+import { Provider } from "./provider";
+import { Address, AppInstanceID } from "./simple-types";
+
+export interface InstallProposalRequest {
```

Comment:
Shouldn't this include the app you're installing?

Reply:
Oh yeah totally, this is still a rough WIP

Reply:
Maybe not? Since the AppFactory will already have the app definition? https://github.com/counterfactual/monorepo/pull/252/files#diff-0ecb04453a0af52b368b562894258f61R37